workshop/exercise-4/README.md

```diff
@@ -1,10 +1,27 @@
-## Deploy Our First Knative Application
-Knative Serving supports deploying and serving of serverless applications and functions. Those applications and functions will automatically scale up, and then back down to zero. In this exercise, we'll use the Knative Serving component to deploy our first application from a container image hosted on dockerhub.
+## Clone the Application Repo and Deploy our Application using Kubectl and yaml
 
-### Create a Service Definition
+### Clone the application repo
+Let's get the code we'll use for today's lab. This repository contains the code for the Fibonacci application as well as various .yaml files we'll use throughout the lab.
+
+1. Clone the git repository:
+
+    ```
+    git clone https://github.com/IBM/fib-knative.git
+    ```
+
+2. Change directories to the fib-knative folder.
+
+    ```
+    cd fib-knative
+    ```
+
+## Deploy Our Application using kubectl and yaml
+In this exercise, we'll use the Knative Serving component to deploy our application from a container image hosted on dockerhub. Instead of using the `kn` cli, we'll use `kubectl` and `.yaml` files. This should feel familiar if you're a kubernetes user.
+
+### Create the Configuration and Route for our Service
 Knative defines some objects for each component as Kubernetes [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources)(CRDs). A CRD is used to define a new resource type in Kubernetes. Knative [Serving](https://github.com/knative/docs/tree/master/docs/serving#serving-resources) includes a number of Custom Resource Definitions, including Service, Route, Configuration, and Revision.
```

Comment:
I like having a little bit of detail in the paragraphs.  I don't think they're really digging that far into it, but they get to pick up a little "behind the scenes" knowledge along the way in case they're interested in learning more.  Based on our last implementation of this lab, people zoomed right through it quite easily, so having a little bit deeper knowledge if they want is not a bad thing IMO.